core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -157,4 +157,30 @@ class TopicCommandTest extends ZooKeeperTestHarness with Logging {
       Array("--partitions", numPartitions.toString, "--replication-factor", "1", "--topic", topic, "--if-not-exists"))
     TopicCommand.createTopic(zkUtils, createNotExistsOpts)
   }
+
+  @Test
+  def testCreateAlterTopicWithRackAware() {
```

Comment:
Similar to the above. Do we need to start ZK in this test? It seems that we can just test AdminUtils.assignReplicasToBrokers() directly?